task.aspx

```diff
@@ -28,16 +35,15 @@
                         <label>Task ID</label>
                         <div class="form-group">
                            <div class="input-group">
-                              <asp:TextBox CssClass="form-control" ID="TextBox5" runat="server" placeholder="Task ID"></asp:TextBox>
-                              <asp:LinkButton class="btn btn-primary" ID="LinkButton4" runat="server"><i class="fas fa-check-circle"></i></asp:LinkButton>
+                              <asp:TextBox CssClass="form-control" ID="txtTaskId" runat="server" placeholder="Task ID"></asp:TextBox>
```

Comment:
### Good naming convention

Good job on naming this and the following items with proper IDs.